### PR TITLE
[Docs] Add docs_zh_CN of TensorRT

### PR DESCRIPTION
**Pull request by grimoire**
## Motivation

Add Chinese document of TensorRT custom ops.

## Modification

Update `docs_zh_CN/deployment/tensorrt_custom_ops.md` and `docs_zh_CN/deployment/tensorrt_plugin.md`

### CODE REVIEW

File: docs_zh_CN/deployment/tensorrt_custom_ops.md [hunk 1]
```diff
@@ -1,3 +1,391 @@
...
+
+### 模型参数
+
+无
+
+### 输入
+
+<dl>
+<dt><tt>inputs[0]</tt>: T</dt>
+<dd>秩为r >= 1的输入`data`。</dd>
```
Comment (zhouzaida) [inline docs_zh_CN/deployment/tensorrt_custom_ops.md:122]:
```suggestion
<dd>秩为r >= 1的输入`data`</dd>
```

File: docs_zh_CN/deployment/tensorrt_custom_ops.md [hunk 2]
```diff
@@ -1,3 +1,391 @@
...
+
+<dt><tt>inputs[1]</tt>: tensor(int32, Linear)</dt>
+<dd>秩为q >= 1的输入`update`。</dd>
```
Comment (zhouzaida) [inline docs_zh_CN/deployment/tensorrt_custom_ops.md:125]:
```suggestion
<dd>秩为q >= 1的输入`update`</dd>
```

File: docs_zh_CN/deployment/tensorrt_custom_ops.md [hunk 3]
```diff
@@ -1,3 +1,391 @@
...
+
+<dt><tt>inputs[2]</tt>: T</dt>
+<dd>秩为 q + r - indices.shape[-1] -1 的输入`update`。</dd>
```
Comment (zhouzaida) [inline docs_zh_CN/deployment/tensorrt_custom_ops.md:128]:
```suggestion
<dd>秩为 q + r - indices.shape[-1] -1 的输入`update`</dd>
```

File: docs_zh_CN/deployment/tensorrt_custom_ops.md [hunk 4]
```diff
@@ -1,3 +1,391 @@
...
+</dl>
+
+### 输出
+
+<dl>
+<dt><tt>outputs[0]</tt>: T</dt>
+<dd>根据描述中方法计算的输出</dd>
```
Comment (zhouzaida) [inline docs_zh_CN/deployment/tensorrt_custom_ops.md:135]: It is not consistent with English version

File: docs_zh_CN/deployment/tensorrt_custom_ops.md [hunk 5]
```diff
@@ -1,3 +1,391 @@
...
+| `int`   | `max_output_boxes_per_class` | 每一类最大的输出检测框个数。默认为0，输出检测框个数等于输入候选框数                      |
+| `float` | `iou_threshold`              | 用来判断候选框重合度的阈值，取值范围[0, 1]。默认值为0                                    |
+| `float` | `score_threshold`            | 用来判断候选框是否合法的阈值                                                             |
+| `int`   | `offset`                     | 检测框长宽计算方式为(x2 - x1 + offset)，可选值0或1                                       |
+
+### 输入
+
+<dl>
+<dt><tt>inputs[0]</tt>: T</dt>
+<dd>输入候选框。形状为(num_batches, spatial_dimension, 4)的三维张量。</dd>
```
Comment (zhouzaida) [inline docs_zh_CN/deployment/tensorrt_custom_ops.md:162]:
```suggestion
<dd>输入候选框。形状为(num_batches, spatial_dimension, 4)的三维张量</dd>
```

File: docs_zh_CN/deployment/tensorrt_custom_ops.md [hunk 6]
```diff
@@ -1,3 +1,391 @@
...
+<dt><tt>inputs[1]</tt>: T</dt>
+<dd>输入得分。形状为(num_batches, num_classes, spatial_dimension)的三维张量。</dd>
```
Comment (zhouzaida) [inline docs_zh_CN/deployment/tensorrt_custom_ops.md:164]:
```suggestion
<dd>输入得分。形状为(num_batches, num_classes, spatial_dimension)的三维张量</dd>
```

File: docs_zh_CN/deployment/tensorrt_custom_ops.md [hunk 7]
```diff
@@ -1,3 +1,391 @@
...
+| `list of ints` | `dilation`         | 卷积核各元素间隔 (dH, dW)                                                                     |
+| `int`          | `deformable_group` | 可变偏移量的分组                                                                              |
+| `int`          | `group`            | 卷积分组数，`input_channel`会根据这个值被分为数个分组进行计算                                 |
+| `int`          | `im2col_step`      | 可变卷积使用im2col计算卷积。输入与偏移量会以im2col_step为步长分块计算，减少临时空间的使用量。 |
+
+### 输入
+
+<dl>
+<dt><tt>inputs[0]</tt>: T</dt>
+<dd>输入特征；形状为(N, C, inH, inW)的四维张量，其中N为batch大小，C为输入通道数，inH和inW为输入特征图的高和宽。</dd>
```
Comment (zhouzaida) [inline docs_zh_CN/deployment/tensorrt_custom_ops.md:201]:
```suggestion
<dd>输入特征；形状为(N, C, inH, inW)的四维张量，其中N为batch大小，C为输入通道数，inH和inW为输入特征图的高和宽</dd>
```

File: docs_zh_CN/deployment/tensorrt_custom_ops.md [hunk 8]
```diff
@@ -1,3 +1,391 @@
...
+<dt><tt>inputs[1]</tt>: T</dt>
+<dd>输入偏移量；形状为(N, deformable_group* 2* kH* kW, outH, outW)的四维张量，kH和kW为输入特征图的高和宽，outH和outW为输入特征图的高和宽。</dd>
```
Comment (zhouzaida) [inline docs_zh_CN/deployment/tensorrt_custom_ops.md:203]:
```suggestion
<dd>输入偏移量；形状为(N, deformable_group* 2* kH* kW, outH, outW)的四维张量，kH和kW为输入特征图的高和宽，outH和outW为输入特征图的高和宽</dd>
```

File: docs_zh_CN/deployment/tensorrt_custom_ops.md [hunk 9]
```diff
@@ -1,3 +1,391 @@
...
+<dt><tt>inputs[2]</tt>: T</dt>
+<dd>输入权重；形状为(output_channel, input_channel, kH, kW)的四维张量。</dd>
```
Comment (zhouzaida) [inline docs_zh_CN/deployment/tensorrt_custom_ops.md:205]:
```suggestion
<dd>输入权重；形状为(output_channel, input_channel, kH, kW)的四维张量</dd>
```

File: docs_zh_CN/deployment/tensorrt_custom_ops.md [hunk 10]
```diff
@@ -1,3 +1,391 @@
...
+</dl>
+
+### 输出
+
+<dl>
+<dt><tt>outputs[0]</tt>: T</dt>
+<dd>输出特征；形状为(N, output_channel, outH, outW)的四维张量。</dd>
```
Comment (zhouzaida) [inline docs_zh_CN/deployment/tensorrt_custom_ops.md:212]:
```suggestion
<dd>输出特征；形状为(N, output_channel, outH, outW)的四维张量</dd>
```

File: docs_zh_CN/deployment/tensorrt_custom_ops.md [hunk 11]
```diff
@@ -1,3 +1,391 @@
...
+| ----- | -------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `int` | `interpolation_mode` | 计算输出使用的插值模式。(0: `bilinear` , 1: `nearest`)                                                                                               |
+| `int` | `padding_mode`       | 边缘填充模式。(0: `zeros`, 1: `border`, 2: `reflection`)                                                                                             |
+| `int` | `align_corners`      | 如果`align_corners=1`，则极值(`-1`和`1`)会被当做输入边缘像素的中心点。如果`align_corners=0`，则它们会被看做是边缘像素的边缘点,减小分辨率对采样的影响 |
+
+### 输入
+
+<dl>
+<dt><tt>inputs[0]</tt>: T</dt>
+<dd>输入特征；形状为(N, C, inH, inW)的四维张量，其中N为batch大小，C为输入通道数，inH和inW为输入特征图的高和宽。</dd>
```
Comment (zhouzaida) [inline docs_zh_CN/deployment/tensorrt_custom_ops.md:237]:
```suggestion
<dd>输入特征；形状为(N, C, inH, inW)的四维张量，其中N为batch大小，C为输入通道数，inH和inW为输入特征图的高和宽</dd>
```

File: docs_zh_CN/deployment/tensorrt_custom_ops.md [hunk 12]
```diff
@@ -1,3 +1,391 @@
...
+<dt><tt>inputs[1]</tt>: T</dt>
+<dd>输入网格；形状为(N, outH, outW, 2)的四维张量，outH和outW为输出的高和宽。 </dd>
```
Comment (zhouzaida) [inline docs_zh_CN/deployment/tensorrt_custom_ops.md:239]:
```suggestion
<dd>输入网格；形状为(N, outH, outW, 2)的四维张量，outH和outW为输出的高和宽 </dd>
```

File: docs_zh_CN/deployment/tensorrt_custom_ops.md [hunk 13]
```diff
@@ -1,3 +1,391 @@
...
+</dl>
+
+### 输出
+
+<dl>
+<dt><tt>outputs[0]</tt>: T</dt>
+<dd>输出特征；形状为(N, C, outH, outW)的四维张量。</dd>
```
Comment (zhouzaida) [inline docs_zh_CN/deployment/tensorrt_custom_ops.md:246]:
```suggestion
<dd>输出特征；形状为(N, C, outH, outW)的四维张量</dd>
```

File: docs_zh_CN/deployment/tensorrt_custom_ops.md [hunk 14]
```diff
@@ -1,3 +1,391 @@
...
+<dd>输入张量；可以使任意形状</dd>
+</dl>
+
+### 输出
+
+<dl>
+<dt><tt>outputs[0]</tt>: T</dt>
+<dd>`input`第`dim`维的累计最大值，形状与`input`相同。类型和`input`一致</dd>
+<dt><tt>outputs[1]</tt>: (int32, Linear)</dt>
+<dd>第`dim`维最大值位置，形状与`input`相同。</dd>
```
Comment (zhouzaida) [inline docs_zh_CN/deployment/tensorrt_custom_ops.md:278]:
```suggestion
<dd>第`dim`维最大值位置，形状与`input`相同</dd>
```

File: docs_zh_CN/deployment/tensorrt_custom_ops.md [hunk 15]
```diff
@@ -1,3 +1,391 @@
...
+<dd>输入张量；可以使任意形状</dd>
+</dl>
+
+### 输出
+
+<dl>
+<dt><tt>outputs[0]</tt>: T</dt>
+<dd>`input`第`dim`维的累计最小值，形状与`input`相同。类型和`input`一致</dd>
+<dt><tt>outputs[1]</tt>: (int32, Linear)</dt>
+<dd>第`dim`维最小值位置，形状与`input`相同。</dd>
```
Comment (zhouzaida) [inline docs_zh_CN/deployment/tensorrt_custom_ops.md:310]:
```suggestion
<dd>第`dim`维最小值位置，形状与`input`相同</dd>
```

File: docs_zh_CN/deployment/tensorrt_custom_ops.md [hunk 16]
```diff
@@ -1,3 +1,391 @@
...
+
+| 类型    | 参数名    | 描述                         |
+| ------- | --------- | ---------------------------- |
+| `float` | `epsilon` | 用来避免除0错误。默认为1e-05 |
+
+### 输入
+
+<dl>
+<dt><tt>inputs[0]</tt>: T</dt>
+<dd>输入特征。形状为(N, C, H， W)的四维张量，其中N为batch大小，C为输入通道数，H和W为输入特征图的高和宽。</dd>
```
Comment (zhouzaida) [inline docs_zh_CN/deployment/tensorrt_custom_ops.md:333]:
```suggestion
<dd>输入特征。形状为(N, C, H， W)的四维张量，其中N为batch大小，C为输入通道数，H和W为输入特征图的高和宽</dd>
```

File: docs_zh_CN/deployment/tensorrt_custom_ops.md [hunk 17]
```diff
@@ -1,3 +1,391 @@
...
+<dt><tt>inputs[1]</tt>: T</dt>
+<dd>输入缩放系数。形状为(C，)的一维张量。</dd>
```
Comment (zhouzaida) [inline docs_zh_CN/deployment/tensorrt_custom_ops.md:335]:
```suggestion
<dd>输入缩放系数。形状为(C，)的一维张量</dd>
```

File: docs_zh_CN/deployment/tensorrt_custom_ops.md [hunk 18]
```diff
@@ -1,3 +1,391 @@
...
+<dt><tt>inputs[2]</tt>: T</dt>
+<dd>输入偏移量。形状为(C，)的一维张量。</dd>
```
Comment (zhouzaida) [inline docs_zh_CN/deployment/tensorrt_custom_ops.md:337]:
```suggestion
<dd>输入偏移量。形状为(C，)的一维张量</dd>
```

File: docs_zh_CN/deployment/tensorrt_custom_ops.md [hunk 19]
```diff
@@ -1,3 +1,391 @@
...
+</dl>
+
+### 输出
+
+<dl>
+<dt><tt>outputs[0]</tt>: T</dt>
+<dd>输出特征。形状为(N, C, H， W)的四维张量。</dd>
```
Comment (zhouzaida) [inline docs_zh_CN/deployment/tensorrt_custom_ops.md:344]:
```suggestion
<dd>输出特征。形状为(N, C, H， W)的四维张量</dd>
```

File: docs_zh_CN/deployment/tensorrt_custom_ops.md [hunk 20]
```diff
@@ -1,3 +1,391 @@
...
+| `list of ints` | `padding`           | 输入特征填充大小 (padH, padW)                                 |
+| `list of ints` | `dilation`          | 卷积核各元素间隔 (dH, dW)                                     |
+| `int`          | `deformable_groups` | 可变偏移量的分组，通常置位1即可                               |
+| `int`          | `groups`            | 卷积分组数，`input_channel`会根据这个值被分为数个分组进行计算 |
+
+### 输入
+
+<dl>
+<dt><tt>inputs[0]</tt>: T</dt>
+<dd>输入特征；形状为(N, C, inH, inW)的四维张量，其中N为batch大小，C为输入通道数，inH和inW为输入特征图的高和宽。</dd>
```
Comment (zhouzaida) [inline docs_zh_CN/deployment/tensorrt_custom_ops.md:371]:
```suggestion
<dd>输入特征；形状为(N, C, inH, inW)的四维张量，其中N为batch大小，C为输入通道数，inH和inW为输入特征图的高和宽</dd>
```

File: docs_zh_CN/deployment/tensorrt_custom_ops.md [hunk 21]
```diff
@@ -1,3 +1,391 @@
...
+<dt><tt>inputs[1]</tt>: T</dt>
+<dd>输入偏移量；形状为(N, deformable_group* 2* kH* kW, outH, outW)的四维张量，kH和kW为输入特征图的高和宽，outH和outW为输入特征图的高和宽。</dd>
```
Comment (zhouzaida) [inline docs_zh_CN/deployment/tensorrt_custom_ops.md:373]:
```suggestion
<dd>输入偏移量；形状为(N, deformable_group* 2* kH* kW, outH, outW)的四维张量，kH和kW为输入特征图的高和宽，outH和outW为输入特征图的高和宽</dd>
```

File: docs_zh_CN/deployment/tensorrt_custom_ops.md [hunk 22]
```diff
@@ -1,3 +1,391 @@
...
+<dt><tt>inputs[2]</tt>: T</dt>
+<dd>输入掩码；形状为(N, deformable_group* kH* kW, outH, outW)的四维张量。</dd>
```
Comment (zhouzaida) [inline docs_zh_CN/deployment/tensorrt_custom_ops.md:375]:
```suggestion
<dd>输入掩码；形状为(N, deformable_group* kH* kW, outH, outW)的四维张量</dd>
```

File: docs_zh_CN/deployment/tensorrt_custom_ops.md [hunk 23]
```diff
@@ -1,3 +1,391 @@
...
+<dt><tt>inputs[3]</tt>: T</dt>
+<dd>输入权重；形状为(output_channel, input_channel, kH, kW)的四维张量。</dd>
```
Comment (zhouzaida) [inline docs_zh_CN/deployment/tensorrt_custom_ops.md:377]:
```suggestion
<dd>输入权重；形状为(output_channel, input_channel, kH, kW)的四维张量</dd>
```

File: docs_zh_CN/deployment/tensorrt_custom_ops.md [hunk 24]
```diff
@@ -1,3 +1,391 @@
...
+<dt><tt>inputs[4]</tt>: T, optional</dt>
+<dd>输入偏移量；形状为(output_channel)的一维张量。</dd>
```
Comment (zhouzaida) [inline docs_zh_CN/deployment/tensorrt_custom_ops.md:379]:
```suggestion
<dd>输入偏移量；形状为(output_channel)的一维张量</dd>
```

File: docs_zh_CN/deployment/tensorrt_custom_ops.md [hunk 25]
```diff
@@ -1,3 +1,391 @@
...
+</dl>
+
+### 输出
+
+<dl>
+<dt><tt>outputs[0]</tt>: T</dt>
+<dd>输出特征；形状为(N, output_channel, outH, outW)的四维张量。</dd>
```
Comment (zhouzaida) [inline docs_zh_CN/deployment/tensorrt_custom_ops.md:386]:
```suggestion
<dd>输出特征；形状为(N, output_channel, outH, outW)的四维张量</dd>
```

**Review by zhouzaida — APPROVED** [review]
LGTM

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**